src/particles/distribution/All.H

```diff
@@ -0,0 +1,23 @@
+/* Copyright 2021 Axel Huebl
```

Comment:
I think this file is not needed.
We can include the existing one (`#include "particles/elements/All.H"`) where we need it.